include/envoy/server/options.h

```diff
@@ -166,6 +166,12 @@ class Options {
    * @return bool indicating whether the hot restart functionality has been disabled via cli flags.
    */
   virtual bool hotRestartDisabled() const PURE;
+
+  /**
+   * @return bool indicating whether or not mutex tracing functionality has been enabled via cli
```

Comment:
@return bool indicating whether mutex tracing functionality has been enabled.

"or not" not necessary.
cli flags --> could also be via setter.